pytest/alignment_test.py

```diff
@@ -0,0 +1,182 @@
+import pytest
+
+import arkouda as ak
+
+
+def test_search_interval():
+    expected_result = [2, 5, 4, 0, 3, 1, 4, -1, -1]
+    lb = [0, 10, 20, 30, 40, 50]
+    ub = [9, 19, 29, 39, 49, 59]
+    v = [22, 51, 44, 1, 38, 19, 40, 60, 100]
+
+    # test int64
+    lower_bound = ak.array(lb)
+    upper_bound = ak.array(ub)
+    vals = ak.array(v)
+    interval_idxs = ak.search_intervals(vals, (lower_bound, upper_bound))
+    assert expected_result == interval_idxs.to_list()

```

Comment:
General question here - Do we lose the ability to run things like `assertListEqual` with this configuration of pytest? Recently, that has become more and more useful for identifying where problems are. If we are still able to use it, we should update this code to use things like that.

Reply:
This is something I've already been looking into. PyTest's failed assertions on lists do provide `diff` output of each list highlighting where problems are, but I have been looking into using custom asserts to handle more complex cases

Reply:
As an example output I purposefully caused a test to fail to show the failure ouput
```
        vals = (ak.array([23, 13, 3]), ak.array([23, 13, 3]))
        ans = [2, 0, 1]
>       assert ans == ak.search_intervals(vals, (starts, ends), hierarchical=False).to_list()
E       assert [2, 0, 1] == [2, 1, 0]
E         At index 1 diff: 0 != 1
E         Use -v to get more diff
```
Adding `-v`
```
>       assert ans == ak.search_intervals(vals, (starts, ends), hierarchical=False).to_list()
E       assert [2, 0, 1] == [2, 1, 0]
E         At index 1 diff: 0 != 1
E         Full diff:
E         - [2, 1, 0]
E         + [2, 0, 1]
```

Reply:
Sounds good. I just wanted to be sure that we weren't losing any information for debugging.